dot/core/service_integration_test.go

```diff
@@ -673,8 +793,9 @@ func TestTryQueryStore_WhenThereIsDataToRetrieve(t *testing.T) {
 	storageStateTrie.Set(testKey, testValue)
 	require.NoError(t, err)
 
+	digest := newTestDigest(t, 420)
```

Comment:
what's `420`?

Maybe have it as an inlined named constant above, like:

```suggestion
	const myNameThatMakesSense = 420
	digest := newTestDigest(t, myNameThatMakesSense)
```

Reply:
updated!